### PR TITLE
Feature/#210refactor transfer batch

### PR DESCRIPTION
**Pull request by HenkKodde**
#210 - Changing transferBatch to transferSettlementBatch V1.0.0.

### CODE REVIEW

**Comment by rmothilal** [comment]
Henk. You request has conflicts that need to be resolved. Also you have committed your change to disable the sidecar. Could you please pull in all changes from develop-PI2 from mojaloop into your branch and see if this resolves you conflicts. Also be careful of the files that you commit

File: config/default.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "SCALE": 2
   },
   "SIDECAR": {
-    "DISABLED": false,
```
Comment (rmothilal) [inline config/default.json]: This needs to remain false

File: src/shared/setup.js [hunk 2]
```diff
@@ -58,9 +58,9 @@ const initialize = async function ({service, port, modules = [], runMigrations =
   await connectDatabase()
   await Sidecar.connect(service)
   const server = await createServer(port, modules)
-  if (service === 'api') {
+  /* if (service === 'api') {
     await Account.createLedgerAccount(Config.LEDGER_ACCOUNT_NAME, Config.LEDGER_ACCOUNT_PASSWORD, Config.LEDGER_ACCOUNT_EMAIL)
```
Comment (rmothilal) [inline src/shared/setup.js]: Now that this is disabled, are we not going to create a default account/participant?

Comment (vgenev) [reply src/shared/setup.js]: Yes, but it was disabled only to pass the 149 story without throwing errors. I will take care of it Monday

Comment (vgenev) [reply src/shared/setup.js]: Yes, but it was disabled only to pass the 149 story without throwing errors. I will take care of it Monday

**Pull request closed without merging**